cli/src/transaction.ts

```diff
@@ -106,6 +106,8 @@ export async function inferTransactions(
       transaction.amount = transaction.useAllAmount
         ? new BigNumber(0)
         : inferAmount(account, opts.amount || "0");
+      // Most probably there's a bug here and this method should `return transaction`,
+      // but every family overwrites this method in cli-transaction.ts so can't really reproduce
```

Comment:
I'm not sure to follow. No bug here to me.

https://github.com/LedgerHQ/ledger-live-common/blob/master/src/families/ethereum/cli-transaction.ts#L101-L109

these have correct types on the other end as it's getting exploded back, but indeed the cli-transaction side will remap them into transaction.

```
function inferTransactions(
  transactions: Array<{
    account: AccountLike;
    transaction: Transaction;
    mainAccount: Account;
  }>,
  opts: Record<string, any>,
  { inferAmount }: any
): Transaction[]
```

Reply:
Ok, removing the comment if it should work like this, thanks :)